ui/raidboss/data/05-shb/raid/e8n.txt

```diff
@@ -4,4 +4,105 @@
+181.9 "--Untargetable--"
+193.2 "Shattered World" sync /:Shiva:4DE9:/
+
+# Intermission. Fixed length.
+209.4 "Heart Asunder" sync /:Mothercrystal:4E12:/
+218.1 "Stoneskin" sync /:Earthen Aether:4DEF:/
+223.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+228.2 "Shock Spikes" sync /:Electric Aether:4DEE:/
+233.2 "Rush" sync /:Luminous Aether:4FC8:/
+233.3 "Rush" sync /:Luminous Aether:4FC9:/
+237.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+251.5 "Heart Asunder" sync /:Mothercrystal:4E12:/
+256.2 "Shock Spikes" sync /:Electric Aether:4DEE:/
+260.2 "Stoneskin" sync /:Earthen Aether:4DEF:/
+261.2 "Rush" sync /:Luminous Aether:4FC8:/
+261.3 "Rush" sync /:Luminous Aether:4FC9:/
+
+# Post-intermission block
+272.6 "Skyfall" sync /:Shiva:4E13:/
+287.9 "--targetable--"
+306.9 "Holy" sync /:Shiva:4DE[CD]:/
+321.4 "Light Rampant" sync /:Shiva:4DE8:/
+329.6 "the Path of Light" sync /:Shiva:4DD9:/
+332.6 "Bright Hunger" sync /:Shiva:4E0C:/
+337.7 "the Path of Light" sync /:Shiva:4DD9:/
+340.6 "Bright Hunger" sync /:Shiva:4E0C:/
+345.9 "the Path of Light" sync /:Shiva:4DD9:/
+348.6 "Bright Hunger" sync /:Shiva:4E0C:/
+356.0 "Mirror, Mirror" sync /:Shiva:4DD4:/
+368.1 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+373.2 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+
+# Rotation block
+385.3 "Redress" sync /:Shiva:4F4E:/
+390.4 "Frost Armor" sync /:Shiva:4DF0:/
+398.4 "Twin Stillness/Twin Silence" sync /:Shiva:4DD[DE]:/
+413.7 "Double Slap" sync /:Shiva:4DDA:/
+424.8 "Mirror, Mirror" sync /:Shiva:4DD4:/
+437.0 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+442.1 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+455.2 "Absolute Zero" sync /:Shiva:4DD7:/
+467.4 "Redress" sync /:Shiva:4F4F:/
+472.6 "Shining Armor" sync /:Shiva:4DF1:/
+480.6 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+484.7 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+500.8 "Holy" sync /:Shiva:4DE[CD]:/
+513.9 "Absolute Zero" sync /:Shiva:4DD7:/
+
+526.1 "Redress" sync /:Shiva:4F4E:/
+531.2 "Frost Armor" sync /:Shiva:4DF0:/
+539.2 "Twin Stillness/Twin Silence" sync /:Shiva:4DD[DE]:/
+554.5 "Double Slap" sync /:Shiva:4DDA:/
+565.6 "Mirror, Mirror" sync /:Shiva:4DD4:/
+577.8 "Kick/Frost" sync /:Shiva:4D(?:D[BC]|E[23]):/
+582.9 "Reflected Kick/Frost" sync /:Frozen Mirror:(?:4DF[EF]|4E0[01]):/
+596.0 "Absolute Zero" sync /:Shiva:4DD7:/
+608.2 "Redress" sync /:Shiva:4F4F:/
+613.4 "Shining Armor" sync /:Shiva:4DF1:/
+621.4 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+625.5 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
+641.6 "Holy" sync /:Shiva:4DE[CD]:/
+654.7 "Absolute Zero" sync /:Shiva:4DD7:/
+
+666.9 "Redress"
+672.0 "Frost Armor"
+680.0 "Twin Stillness/Twin Silence"
+695.3 "Double Slap"
+706.4 "Mirror, Mirror"
+
+# There's an Absolute Zero spam sequence before this, but
+# it's indistinguishable from the standard usage.
+1000.0 "--sync--" sync /:Shiva:4E29:/ window 1000,0
```

Comment:
Also, what's weird is that today I saw:
```
369.0 "Scythe Kick" sync /:Shiva:4DE3:/
374.1 "Reflected Scythe Kick" sync /:Frozen Mirror:4E01:/
386.2 "Redress" sync /:Shiva:4F4E:/
391.3 "--sync--" sync /:Shiva:4E18:/
391.3 "Frost Armor" sync /:Shiva:4DF0:/
398.3 "Redress" sync /:Shiva:4F4F:/  # <--- ?????
403.4 "Shining Armor" sync /:Shiva:4DF1:/  # <--- ?????
411.4 "Absolute Zero" sync /:Shiva:4DD7:/
419.5 "Absolute Zero" sync /:Shiva:4DD7:/
422.7 "--sync--" sync /:Shiva:4D58:/
424.0 "Absolute Zero" sync /:Shiva:4E28:/
428.9 "--sync--" sync /:Shiva:4E29:/
```

Does that mean there's always a redress -> shining armor before the 4DD7 enrage too? I'm not sure this helps anything, but just wanted to mention.

Reply:
Unfortunately that doesn't help here, no, since it's absolutely identical. And yes, she forcefully Redresses if she's not already in Ryne form before enrage.

Reply:
Actually, thinking about it, we *might be able to do something really messy like...

```
385.3 "Redress" sync /:Shiva:4F4E:/
390.4 "Frost Armor" sync /:Shiva:4DF0:/
398.4 "Twin Stillness/Twin Silence" sync /:Shiva:4DD[DE]:/
413.7 "Double Slap" sync /:Shiva:4DDA:/
424.8 "Mirror, Mirror" sync /:Shiva:4DD4:/
437.0 "Kick/Frost" sync /:Shiva:4D(D[BC]|E[23]):/
442.1 "Reflected Kick/Frost" sync /:Frozen Mirror:(4DF[EF]|4E0[01]):/
450.0 [Hidden Redress goes here] sync /:Shiva:4F4F:/ window 70,0 jump to enrage
455.2 "Absolute Zero" sync /:Shiva:4DD7:/
467.4 "Redress" sync /:Shiva:4F4F:/
472.6 "Shining Armor" sync /:Shiva:4DF1:/
480.6 "Embittered Dance/Spiteful Dance" sync /:Shiva:4DE[4567]:/
484.7 "Spiteful Dance/Embittered Dance" sync /:Shiva:4DE[4567]:/
500.8 "Holy" sync /:Shiva:4DE[CD]:/
505 [Hidden Absolute Zero goes here] sync /:Shiva:4DD7:/ window 100,0 jump to enrage
513.9 "Absolute Zero" sync /:Shiva:4DD7:/
```

It's a hack, and it has a small chance of failure.

Reply:
Yeah, I think that's plausible.  Maybe let's get this patch merged, and this can be left as a TODO.

Reply:
Okay. Are you comfortable if I file Yet Another Issue so I don't forget?

Reply:
Yeah, an issue is fine!